hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/common/Call.java

```diff
@@ -150,4 +151,15 @@ default PricedResult execute(MessageFrame frame) {
     default boolean allowsStaticFrame() {
         return false;
     }
+
+    /**
+     * Prepares a {@link SchedulableTransactionBody} for dispatching a scheduled call.
+     *
+     * @param frame the message frame
+     * @return the native TransactionBody implied by this call
+     */
+    @NonNull
+    default SchedulableTransactionBody asSchedulableDispatchIn(@NonNull final MessageFrame frame) {
```

Comment:
I believe that the `MessageFrame` is not needed anywhere.

Reply:
I was wondering if we'll need something from it for other scheduled cases but we can overload it if needed at later stage.